content/storage-history/storage.textile

```diff
@@ -0,0 +1,109 @@
+---
+title:  Message Storage
+meta_description: "Explore the different ways Ably can handle Message Storage"
+
+---
+
+By default Ably will store messages for just two minutes so that they can be retrieved with the "History API":/storage-history/history in the event of a lost connection. However, there are a number of options available to increase this storage time. You can persist all of your messages (typically for 24 hours), persist the last message using the "channel rewind":/channels/options/rewind option, or use "integrations":/integrations to persist messages externally. 
+
+
+h2(#default-persistence). Default message storage - two minutes
+
+Accounts with the default message storage of two minutes can use the History API to recover messages in the original order they were sent, and applies to both "regular channel messages":/realtime/messages and "presence messages":/realtime/presence. For brief disconnections Ably automatically sends lost messages via the live subscription.
+
+The following diagram illustrates the default persistence of messages:
+
+<a href="/images/diagrams/history-default.png" target="_blank">
+  <img src="/images/diagrams/history-default.png" style="width: 100%" alt="Default Persistence">
+</a>
+
+Read more about the "History API":/storage-history/history or the "Realtime connection state recovery":/realtime/connection/#connection-state-recovery.
+
+
+h2(#all-message-persistence). Persist all messages - 24 or 72 hours
+
+If you need to retain messages for longer than the default two minutes you can enable "persisted history" when setting a "channel rule":/general/channel-rules-namespaces. When "persisted history is enabled":#persisted-history for a channel any messages will be stored on disk for 24 for free tiers or 72 hours for paid tiers.
+
+In this scenario, the History API can be used to retrieve the persisted messages. This is illustrated in the following diagram:
+
+<a href="/images/diagrams/history-persist-all-messages.png" target="_blank">
+  <img src="/images/diagrams/history-persist-all-messages.png" style="width: 100%" alt="Persist All Messages">
+</a>
+
+Every message that is persisted to, or retrieved from, disk counts as an extra message towards your monthly quota. For example, with persistence enabled a published message counts as two messages for your monthly quota. If the message is then retrieved another message will be deducted from your monthly quota.
+
+To enable history on a channel add a @persist all messages@ "channel rule":/general/channel-rules-namespaces in the settings of your Ably "dashboard":https://ably.com/dashboard.
+
+Foldable box
+
+    Select the settings tab from your Ably "dashboard":https://ably.com/dashboard :
+
+    <a href="/images/screenshots/storage/dashboard-settings.png" target="_blank">
```

Comment:
Great job on setting up these screenshots, but can we leave them out of the PR for now until we know the final specs for the UI implementation?